README.md

```
@@ -0,0 +1,58 @@
+
+Land the lunar module (represented by 'A') on a flat surface with a sufficiently low velocity. Watch your fuel and vertical/horizontal speed!
+
+## Development
+
+Based on the [specification](spec.md).
+
+[See the plan](plan.md) for development roadmap.
+
+## License
+
+MIT
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)